src/main/java/net/snowflake/ingest/streaming/internal/ParquetTypeGenerator.java

```diff
@@ -4,13 +4,16 @@
 
 package net.snowflake.ingest.streaming.internal;
 
+import static net.snowflake.ingest.utils.IcebergDataTypeParser.deserializeIcebergType;
+
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 import net.snowflake.ingest.utils.ErrorCode;
 import net.snowflake.ingest.utils.SFException;
+import org.apache.iceberg.parquet.TypeToMessageType;
 import org.apache.parquet.schema.LogicalTypeAnnotation;
 import org.apache.parquet.schema.PrimitiveType;
 import org.apache.parquet.schema.Type;
```

Comment:
nit: please move ParquetTypeInfo to a separate file.

Reply:
Moved.